boa/src/lib.rs

```diff
@@ -31,7 +29,12 @@ This is an experimental Javascript lexer, parser and compiler written in Rust. C
     future_incompatible,
     nonstandard_style,
 )]
-#![warn(clippy::perf, clippy::single_match_else, clippy::dbg_macro)]
+#![warn(
+    clippy::all,
```

Comment:
The problem with this is that even the ones considered as "error" by clippy will be silenced. Maybe the solution is to just not mention them (that way, errors will be errors, warnings will be warnings).

Reply:
But if we exclude it the ones that are neither error or warnings wont we shown...

Maybe leave it as always error and add or change the clippy CI to nightly so we catch them ahead of time and it wont fail on stable or nighlty

Reply:
...Or we could just leave it as it was.